src/components/Layouts/ReportLayout.less

```diff
@@ -37,35 +42,5 @@ h1 {
       width: 100%;
       display: inline-block !important;
     }
-
-  .grid-layout {
```

Comment:
something is strange... I can't find this definition in the master code:

![image](https://user-images.githubusercontent.com/5429033/34887628-cba43dba-f7cf-11e7-8110-e74f4187fc21.png)

Reply:
thats strange.. are you on latest master? looks like old code